src/connector/src/sink/remote.rs

```diff
@@ -70,8 +71,8 @@ use crate::sink::{
 macro_rules! def_remote_sink {
     () => {
         def_remote_sink! {
-            { ElasticSearch, ElasticSearchSink, "elasticsearch" }
-            { Opensearch, OpenSearchSink, "opensearch"}
+            { ElasticSearchJava, ElasticSearchJavaSink, "elasticsearch_v1" }
+            { OpensearchJava, OpenSearchJavaSink, "opensearch_v1"}
```

Comment:
May I ask why to keep `_v1`?

Reply:
Since there are some users using java es, I'd like to be able to keep him for a while, in case something goes wrong with the migration to v2, so that they can use v1 first.

Reply:
To clarify,
- For users already using `connector = elasticsearch`, when they upgrade, they will switch to v2 immediately.
- If they meet problems, they can manually create new sinks with `elasticsearch_v1` as a temporary workaround.

Is this what you want?

The main concern for me is that when we offer `elasticsearch_v1`, it might be hard to delete the legacy implementation in the future. (It might be possible if we only make it an undocumented feature and tell only some users about it...)

Reply:
Yes yes, I'm leaning towards,Only problems that are difficult to solve with v2 will make users try v1, and no updates will be made after v1, he may just exist for a version or two as a transition

Reply:
For existing users, if they encounter any problem in this new rust implementation, instead of letting user recreate a sink with connector as `_v1`, I think we can make a patch and build a separate image to change back to use the java implementation for that users, and meanwhile investigate the fix the rust implementation.

If this is feasible, we can exclude the `_v1` from the sink list.